pages/byoc/installation.mdx

```diff
@@ -0,0 +1,109 @@
+# BYOC Installation
+
+We offer two installation methods, depending on whether you are using kubernetes or vm. Both are very simple and you can deploy in less than 1 minute! they all need agent token, you can get it from ModelZ dashboard.
+![agent_token](./images/agent_token.jpg)
+## VM
+
+### Preparation
+```shell
+$ pip install openmodelz
+# Check install openmodelz correctly
+$ mdz server start --help
+Start the server with the public IP of the machine. If not provided, the internal IP will be used automatically.
+
+Usage:
+  mdz server start [flags]
+
+Examples:
+  mdz server start
+  mdz server start -v
+  mdz server start 1.2.3.4
+
+Flags:
+  -g, --force-gpu                          Start the server with GPU support (ignore the GPU detection)
+  -h, --help                               help for start
+      --mirror-endpoints https://quay.io   Mirror URL endpoints of the registry like https://quay.io
+      --mirror-name string                 Mirror domain name of the registry (default "docker.io")
+      --modelzcloud-agent-token string     ModelZ Cloud Agent Token
+      --modelzcloud-enabled                Enable ModelZ Cloud Management
+      --modelzcloud-region string          ModelZ Cloud Region (default "on-premises")
+      --modelzcloud-url string             ModelZ Cloud URL (default "https://cloud.modelz.ai")
+
+Global Flags:
+      --debug               Enable debug logging
+      --disable-telemetry   Disable anonymous telemetry
+  -u, --url string          URL to use for the server (MDZ_URL) (default http://localhost:80)
+  -v, --verbose             Verbose output
+```
+
+### Installation
+```shell
+$ mdz server start --modelzcloud-agent-token=byoc:cd4a928c-7c66-4934-beb2-98dd82d672ce:b1485312db327bd52a0f256872c090be2c11d451acc5f6ad9f473a181xxxxxx --modelzcloud-enabled=true --modelzcloud-region=us-central1-a 34.134.190.132
+```
+- `--modelzcloud-agent-token` is generated from ModelZ dashboard.
+- `--modelzcloud-enabled` means enable ModelZ Cloud Management.
+- `--modelzcloud-region=us-central1-a` is the region of your owner VPC.
```

Comment:
Do it have a default value? Or maybe we would tell user we only have one region of ModelZ cloud now and its value could only be `us-central1-a`.

Reply:
the region is your own VPC not modelz region.